test/built-ins/Array/prototype/group/groupLength.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 Ecma International.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-array.prototype.group
+description: Array.prototype.group populates object with correct keys and values
```

Comment:
Same here, I'd suggest something like:
```suggestion
description: Callback can return numbers that are converted to property keys
```